apiserver/plane/bgtasks/export_task.py

```
@@ -307,51 +312,116 @@ def issue_export_task(provider, workspace_id, project_ids, token_id, multiple, s
+                "target_date": issue.target_date,
+                "state_name": issue.state.name if issue.state else None,
+                "created_at": issue.created_at,
+                "updated_at": issue.updated_at,
+                "completed_at": issue.completed_at,
+                "archived_at": issue.archived_at,
+                "module_name": [
+                    module.module.name for module in issue.issue_module.all()
+                ]
+                if issue.issue_module.exists()
+                else None,
+                "created_by": f"{issue.created_by.first_name} {issue.created_by.last_name}",
```

Ruff (0.8.2)

Line too long (92 > 88)

(E501)